### PR TITLE
♻️ Refactor styles to be bem like

### PR DESCRIPTION
**Pull request by phlgr**
Use bem naming convention:
https://en.bem.info/methodology/naming-convention/
Create React Components for Icon.

### CODE REVIEW

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Review by lmachens — APPROVED** [review]
o(*￣▽￣*)ブ

File: src/app/components/Icons/SearchFieldIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function SearchIcon(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```
Comment (lmachens) [inline src/app/components/Icons/SearchFieldIcon.tsx]:
```suggestion
function SearchFieldIcon(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```

**Review by lmachens — CHANGES_REQUESTED** [review]

**Review by lmachens — APPROVED** [review]
╰(*°▽°*)╯

**Pull request merged**